__mocks__/@grafana/ui.tsx

```diff
@@ -47,6 +39,5 @@ module.exports = {
   ...ui,
   Icon,
   BigValue,
-  LinkButton,
```

Comment:
Removing this as I wanted to assert on the `aria-disabled` attribute of LinkButton to ensure if the gcom api was unavailable that checks could still be created.

It resulted in a few non-related tests in this PR having to be modified which I think is entirely acceptable.